src/main/java/de/tum/in/www1/artemis/service/connectors/BitbucketService.java

```diff
@@ -296,11 +281,9 @@ private String getProjectKeyFromUrl(URL repositoryUrl) throws BitbucketException
     public String getRepositorySlugFromUrl(URL repositoryUrl) throws BitbucketException {
         // https://ga42xab@repobruegge.in.tum.de/scm/EIST2016RME/RMEXERCISE-ga42xab.git
         String[] urlParts = repositoryUrl.getFile().split("/");
-        if (urlParts.length > 3) {
-            String repositorySlug = urlParts[3];
-            if (repositorySlug.endsWith(".git")) {
-                repositorySlug = repositorySlug.substring(0, repositorySlug.length() - 4);
-            }
+        if (urlParts[urlParts.length - 1].endsWith(".git")) {
```

Comment:
Breaks if we use other URLs in our tests, that don't have the Bitbucket format (which is sometimes the case because we don't only write specific Bitbucket tests, but some also just want to validate the generic VCS service functionality)